.eslintrc.cjs

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+	extends: ['./node_modules/@balena/lint/config/.eslintrc.js'],
+	parserOptions: {
+		project: 'tsconfig.js.json',
+		sourceType: 'module',
+	},
+	env: {
+		mocha: true,
```

Comment:
Since that's the standard way of doing it in es-lint, and our codebase is primarily in TS anyway, I'm fine w/ this as and don't think it's crucial to move it to balena-lint itself, especially since some project might be using a different test lib (I wouldn't be against it either though).